packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -519,6 +519,8 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
     });
   };
 
+  const { onClick: onInputClick, ...inputProps } = getInputProps();
```

Comment:
This onClick prop fork seems to defeat the purpose of having `getInputProps()`.  All the props are supposed to land on the same DOM element.